### PR TITLE
Fix/monorepo

### PR DESCRIPTION
**Pull request by 2bad2furious**
Fixes lokse
Adds quality of life improvements - emits ts declarations of ui package

### CODE REVIEW

File: template/packages/ui/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -1,3 +1,7 @@
-export const Button = () => {
-    return <button>Button</button>;
+interface ButtonProps {
+    disabled?: boolean;
+}
+
+export const Button = ({disabled = false}: ButtonProps) => {
```
Comment (cermakjiri) [inline template/packages/ui/src/components/Button/Button.tsx]: Do you have enabled prettier on save in your IDE?

File: template/apps/web/lokse.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { createLokseConfig } = require('../../packages/localization/lokse/index.cjs');
+
+const { languages } = require('./src/modules/intl/config/langs.cjs');
+
+// FIXME: test yarn lokse
```
Comment (cermakjiri) [inline template/apps/web/lokse.config.js]: No need for the comment I guess :D

File: template/tooling/eslint/nextjs.cjs [hunk 3]
```diff
@@ -15,6 +15,15 @@ const config = {
...
         'react/react-in-jsx-scope': 'off',
+        'react/self-closing-comp': ['warn'],
+        'react/jsx-curly-brace-presence': [
+            'warn',
+            {
+                props: 'never',
+                children: 'never',
+                propElementValues: 'always'
+            }
+        ],
```
Comment (cermakjiri) [inline template/tooling/eslint/nextjs.cjs:26]: 👌

**Review by cermakjiri — APPROVED** [review]

**Comment by cermakjiri** [comment]
Nice 👍  Just minor thing, the code formatting is weird like there's no prettier active.

**Comment by 2bad2furious** [comment]
> Nice 👍 Just minor thing, the code formatting is weird like there's no prettier active.

Honestly had a hard time getting prettier (and ESLint for that matter) working. Both kept complaining about either missing file extension in config or weird .mjs/.js compatibility

File: template/packages/ui/src/components/Button/Button.tsx (hunk 1 above)
Comment (2bad2furious) [reply template/packages/ui/src/components/Button/Button.tsx]: Is it ok now?

File: template/tooling/prettier/package.json [hunk 4]
```diff
@@ -10,6 +10,7 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "prettier": "^3.0.3",
```
Comment (cermakjiri) [inline template/tooling/prettier/package.json]: This won't do much because the `@ackee/prettier-config` already includes prettier as dep.

**Pull request merged**